addOns/reports/src/main/javahelp/org/zaproxy/addon/reports/resources/help/contents/report-traditional-json-plus.html

```diff
@@ -33,12 +33,21 @@ <H3>About riskdesc</H3>
 	<code>High (Medium)</code>
 	would indicate a High risk issue identified with Medium confidence.
 
+	<H4>About created</H4>
```

Comment:
This is again a case where it would be better to mention in a single place and reference rather than quadruplicate the same description.